ipc/src/win.rs

```diff
@@ -292,3 +292,16 @@ impl<M: Metadata, S: Middleware<M> + 'static> Drop for Server<M, S> {
 		trace!(target: "ipc", "IPC Server : shutdown");
 	}
 }
+
+
+pub fn server<I, M: Metadata, S: Middleware<M> + Send + Sync + 'static>(
```

Comment:
`Send + Sync + 'static` bounds are not necessary here, they are part of `Middleware` trait: https://github.com/ethcore/jsonrpc/blob/b36a57c4bf449c431dc59f0e88236026eda62ea7/core/src/middleware.rs#L8